documentation/slate/source/includes/disruptions.md

```diff
@@ -0,0 +1,62 @@
+Disruptions
+===========
+
+
+In this paragraph, we will know how the disruptions are displayed in the different APIs
+
+# Context
+
+## periods
+
+### production period 
+The production period is global to a navitia coverage and is the validity period of the coverage's data.
+This production period cannot excede one year.
+
+### publication period
+The publication period of a disruption is the period on which we want to display the disruption in navitia.
+
+The creator of the disruption might not want the traveller to know of a disruption before a certain date (because it's too uncertain, secret, ...)
+The publication period is the way to control this
+
+### application periods
+The application periods are the list of periods on which the disruption is active
+
+## query dates
+
+Another thing that can be tricky to understand this that there can be several datetimes for a query
+
+### now
```

Comment:
don't you also need to talk about the action period ? else the line below `query dates` is not usefull